README.md

```diff
@@ -87,4 +87,35 @@ After installing the dependencies, clone the repository and run the following co
 make setup
 ```
 
-This will fetch the third party dependencies and build the project, including the Swift bindings.
\ No newline at end of file
+This will fetch the third party dependencies and build the project, including the Swift bindings.
+
+### Devloop
+
+During normal development you can use the `just devloop` command to test your code both during development and before comitting/pushing. This is handy as it runs as many checks as possible and fixes any issues (such as formatting) automatically.
+
+This command does not require any configuration.
```

Comment:
We need to add `just` and `pnpm` to the list of dependencies above ☝️ we should possibly just (no pun intended) remove the `Makefile` and move all the commands to the `justfile`

Reply:
Added in 5831cbaeae4ad76ef4a5c9988afbab562ab2713f

> we should possibly just (no pun intended) remove the Makefile and move all the commands to the justfile

Yeah that would be nice